lib/osut/utils.rb

```diff
@@ -4308,13 +4488,12 @@ def genAnchors(s = nil, set = [], tag = :vtx)
 
   ##
   # Generates extended polygon vertices to circumscribe one or more sets
-  # (Hashes) of sequenced vertices. The method neither alters the original
-  # polygon vertices, nor validates individual sets of vertices (e.g.
-  # non-coplanarity, self-intersecting, inter-set conflicts). Valid leader line
-  # anchors (set key :ld) need to be generated prior to calling the method (see
-  # genAnchors). By default, the method seeks to link leader line anchors to
-  # set :vtx (key) vertices (users can select another collection of vertices,
-  # e.g. tag == :box).
+  # (Hashes) of sequenced vertices. The method minimally validates individual
+  # sets of vertices (e.g. coplanarity, non-self-intersecting, no inter-set
+  # conflicts). Valid leader line anchors (set key :ld) need to be generated
+  # prior to calling the method (see genAnchors). By default, the method seeks
+  # to link leader line anchors to set :vtx (key) vertices (users can select
+  # another collection of vertices, e.g. tag == :box).
```

Comment:
Leader line anchors are needed to safely circumscribe skylight well inserts.
<img width="942" alt="office_wells" src="https://github.com/rd2/osut/assets/98104985/de037559-084f-464d-a93b-48bed7d8960a">
... if safely deployed, the integration can be seamless.
<img width="977" alt="office_skies" src="https://github.com/rd2/osut/assets/98104985/5cc33d8f-f1da-4cd3-a99b-5f0bcde635c8">